.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    versioning-strategy: "auto"
+    open-pull-requests-limit: 1
+    schedule:
+      interval: "weekly"
+    groups:
+      npm-dependencies:
+        patterns:
+          - "*"
+        update-types:
+          - "minor"
+          - "patch"
+  - package-ecosystem: "pip"
+    directory: "/requirements"
+    versioning-strategy: "auto"
+    open-pull-requests-limit: 1
+    schedule:
+      interval: "weekly"
+    groups:
+      pypi-dependencies:
+        patterns:
+          - "*"
+        exclude-patterns:
+          - "ci.txt"
```

Comment:
From https://github.com/maykinmedia/open-api-framework/issues/52:

> Excluding ci.txt: We have excluded ci.txt from the pip dependencies because there is no matching ci.in file. Ensure that both .txt and .in are present if you want them to be parsed (only works for version updates - security updated still flaky).